### PR TITLE
Create snippets.json

### LINKED ISSUE #13: [FEATURE] Create a a file called `snippets.json` which will list all the projects
Example for an object in the file
```json
[
  {
    "name": "Cart Item",
    "codepen-link": "https://codepen.io/deveesh_shetty_12/pen/rNKXvKm",
    "image-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/assets/89470104/9b12a923-f70a-411e-8964-0273e507576a",
    "folder-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/tree/master/Cart%20Item",
    "author": "Deveesh-Shetty"
  }
]
```

You can get all these details from the `Readme.md` file for all the available sniipets

## Why this?
Because I want to have a frontend for this project, it will be easier to access all the data from the JSON file and put it in the UI. 

Feel free to ask questions about it!

### PR DESCRIPTION
**Pull request by Nilothpal-Pillai**
Fixes #13 : Created snippets.json file with info of all projects

### CODE REVIEW

File: snippets.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    "folder-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/tree/master/UI%20Snippets/Chart/BarChart",
+    "author": "Deveesh-Shetty"
+  },
+  {
+    "name": "Loading Page",
+    "codepen-link": "https://codepen.io/ahmedelmsery/pen/zZVLMN",
+    "image-link": "https://res.cloudinary.com/duwcigilo/image/upload/v1684336253/Loading_image_ig5hh7.jpg",
+    "folder-link": "https://github.com/Deveesh-Shetty/CSS-Is-Fun/tree/master/UI%20Snippets/Loading%20Page",
+    "author": "Deveesh-Shetty"
+  },
```
Comment (Deveesh-Shetty) [inline snippets.json:92]: The author for this snippet is someone else check the codepen link and author name from #8

**Review by Deveesh-Shetty — CHANGES_REQUESTED** [review]
The author

File: snippets.json (hunk 1 above)
Comment (Nilothpal-Pillai) [reply snippets.json]: Changes have been made, please review again.

**Review by Deveesh-Shetty — APPROVED** [review]
Thank you for the contribution!

**Pull request merged**